vars/bayesianScanner.groovy

```diff
@@ -14,10 +14,14 @@ def call(body) {
         echo "Checking ${serviceName} exists"
         if (flow.hasService(serviceName)) {
             try {
-                sh 'mvn io.github.stackinfo:stackinfo-maven-plugin:0.2:prepare'
+                sh '''
+                    mvn io.github.stackinfo:stackinfo-maven-plugin:0.2:prepare &&
```

Comment:
@yzainee can we drop `&` from commands for better readability. I think it could also work. with `&`

Reply:
@hrishin no, `&`  and `&&` has different meaning in bash.

Reply:
`x && y && z ` will execute ` y` only if `x` exited with `0` and `z ` will be executed only if `y` exited with `0` 

So I am fine with the current patch but it needs to be tested before merging. I also need to check if sh would terminate if any of the commands fail so that we can completely avoid `&&`

Reply:
I see @sthaha :+1: . Without `&&` it would work the same way. Isn't it?

Reply:
@yzainee @hrishin yes, .. I just tested this on my local jenkins. You can avoid the `&&` completely 

if any of the command in `sh` fails the next will not be executed and the build would be marked as failed. e..g

```groovy
sh """
   echo first command
   ls /adalfjdsalf
   echo last command
"""
```
will stop execution when `  ls /adalfjdsalf` fails.